configs/milvus.yaml

```diff
@@ -979,6 +979,14 @@ quotaAndLimits:
       enabled: false # switch to enable l0 segment row count quota
       lowWaterLevel: 32768 # l0 segment row count quota, low water level
       highWaterLevel: 65536 # l0 segment row count quota, low water level
+    deleteBufferRowCountProtection:
+      enabled: false # switch to enable delete buffer row count quota
+      lowWaterLevel: 32768 # delete buffer row count quota, low water level
+      highWaterLevel: 65536 # delete buffer row count quota, high water level
```

Comment:
This can be merged ahead, but the waterlevel is not a sensible one, need to adjust it before put in prod

Reply:
agree, this value shall be updated after we get some experience data from the tests